traffic_ops/traffic_ops_golang/deliveryservice/eligible.go

```diff
@@ -86,13 +105,26 @@ s.ilo_ip_gateway,
 s.ilo_ip_netmask,
 s.ilo_password,
 s.ilo_username,
-COALESCE(s.interface_mtu, ` + strconv.Itoa(JumboFrameBPS) + `) as interface_mtu,
-s.interface_name,
-s.ip6_address,
-s.ip6_gateway,
-s.ip_address,
-s.ip_gateway,
-s.ip_netmask,
+	ARRAY (
+SELECT ( json_build_object (
```

Comment:
So this does worry me a bit still. When reviewing @ocket8888's PR I found that this query using `json_build_object` is like 12x slower than querying the interfaces and IPs separately (without using `json_build_object`) and manually populating the servers structs with that data. Also, we should really be making the same query here as we would in the /servers API. It seems like we could make this better by sharing that code/query here.

Locally (which I typically assume performs much better than in Prod), for a DS w/ ~1000 servers assigned, this API takes 850ms for me, whereas before this PR, it takes ~75ms.

Reply:
True enough. This is also still how `/servers/details` is being done.